src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -114,15 +114,15 @@ export const TxHistoryNavigator = () => {
         <Stack.Screen name="select-asset" options={{title: strings.selectAssetTitle}}>
           {({navigation}: {navigation: TxHistoryRouteNavigation}) => (
             <AssetSelectorScreen
-              assetTokens={tokenBalance.values}
-              onSelect={(token) => {
+              balance={balance}
```

Comment:
balances

Reply:
it was fixed in a previous commit. Thanks